woocommerce/google-listings-and-ads
# Adding Attribute Mapping Page under Settings

### Changes proposed in this Pull Request:

Closes part of #1648.

This PR creates Attribute mapping page under Settings tab.

Figma: fqR0EHi63lWahRcVTKCcba-fi-883%3A114583

### Screenshots:

<img width="1792" alt="Screenshot 2022-09-02 at 15 44 19" src="https://user-images.githubusercontent.com/5908855/188132611-661190ac-13ec-4f38-b5f3-dfe043bafb70.png">

### Detailed test instructions:

1. Complete an onboarded setup and see the new tab Attribute Mapping i
3. Click on Attributes Mapping Tab and see the new Attribute Mapping page
5. In  Attribute Mapping check that UI looks similar to Figma design
6. Enter in `/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fattribute-mapping`  and see that you can also see the Attributes mapping tab
